docs/FAQ.md

```diff
@@ -102,6 +102,14 @@ hub pull someone_else
 
 It is not always straightforward to tell `gem` to look in non-standard directories for headers and libraries. If you choose `/usr/local`, many things will "just work".
 
+## Why does Homebrew on Linux recommend that I install to `/home/linuxbrew/.linuxbrew`?
+
+The precompiled binary bottles of non-relocatable bottles can only be used if you install in `/home/linuxbrew/.linuxbrew`, otherwise they have to be built from source. Install Homebrew on Linux in `/home/linuxbrew/.linuxbrew/` if possible so that you can use precompiled binary packages (known as bottles) for non-relocatable formula. If you do not yourself have admin privileges, consider asking your admin staff to create a `linuxbrew` role account for you with home directory `/home/linuxbrew`.
```

Comment:
Even as a maintainer I barely understand this sentence (relocatable, non-relocatable) so I'm not sure that end users would.

Maybe:

```suggestion
Homebrew's pre-built binary packages of most software can only be used if you install in `/home/linuxbrew/.linuxbrew`, otherwise they have to be built from source. Building from source can take a long time, is error-prone and isn't as well supported as using our binaries. Install Homebrew on Linux in `/home/linuxbrew/.linuxbrew/` if possible so that you can use our pre-built binary packages (known as bottles) for formulae. If you do not yourself have admin privileges, consider asking your admin staff to create a `linuxbrew` role account for you with home directory `/home/linuxbrew`.
```